addons/l10n_it_edi_website_sale/views/templates.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="address_b2b" inherit_id="website_sale.address_b2b" name="Show l10n_it fields" customize_show="True">
+            <xpath expr="//div[hasclass('div_vat')]" position="after">
```

Comment:
I guess `hasclass` doesn't work with `t-attf-class` cf. https://github.com/odoo/odoo/blob/10cf2aec56c81a7639a435df3d2120e6f62dea0d/addons/website_sale/views/templates.xml#L1502

A good test would have catches that.

```suggestion
            <xpath expr="//div[contains(@t-attf-class, 'div_vat')]" position="after">
```

Reply:
yep, i didn't test it I tought it's good since the tests passed, sorry